### PR TITLE
Birdeye view for carla_spectator

### PR DESCRIPTION
**Pull request by emailweixu**
Solve #626

### CODE REVIEW

File: alf/environments/carla_spectator.py [hunk 1]
```diff
@@ -60,9 +60,9 @@ def main(_):
     pygame.init()
     pygame.font.init()
 
-    width = 480
-    height = 240
-    display = pygame.display.set_mode((width, height),
+    width = 640
+    height = 320
+    display = pygame.display.set_mode((640, 320),
```
Comment (le-horizon) [inline alf/environments/carla_spectator.py]: Should this line still use (width, height)?

**Review by le-horizon — DISMISSED** [review]
LG, Wei.  Please check comment to see if it makes sense.

Thanks,
Le

**Comment by emailweixu** [comment]
@le-horizon need your approval again after addressing your comment

**Review by le-horizon — APPROVED** [review]
LG

**Pull request merged**